api/src/pcapi/routes/backoffice/offer_edition_rules/blueprint.py

```diff
@@ -0,0 +1,188 @@
+        if form.category.data:
+            query = query.filter(
+                offers_models.OfferEditionRule.subcategoryId.in_(
+                    subcategory.id
+                    for subcategory in subcategories_v2.ALL_SUBCATEGORIES
+                    if subcategory.category.id in form.category.data
+                )
+            )
+
+    rules = query.all()
+
+    return render_template(
+        "offer_edition_rules/list_rules.html",
+        rows=rules,
+        form=form,
+        dst=url_for(".list_rules"),
+    )
+
+
+@offer_edition_rules_blueprint.route("/create", methods=["GET"])
+def get_create_offer_edition_rule_form() -> utils.BackofficeResponse:
+    form = forms.CreateOfferEditionRuleForm()
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=form,
+        dst=url_for("backoffice_web.offer_edition_rules.create_rule"),
+        div_id="create-offer-edition-rule",  # must be consistent with parameter passed to build_lazy_modal
+        title="Créer une règle de modification d'offre",
+        button_text="Créer la règle",
+    )
+
+
+@offer_edition_rules_blueprint.route("/create", methods=["POST"])
+def create_rule() -> utils.BackofficeResponse:
+    form = forms.CreateOfferEditionRuleForm()
+
+    if not form.validate():
+        flash(utils.build_form_error_msg(form), "warning")
+        return redirect(url_for("backoffice_web.offer_edition_rules.list_rules"), code=303)
+
+    rate = Decimal(form.rate.data / 100).quantize(Decimal("0.0001"))
+    try:
+        new_rule = offers_models.OfferEditionRule(subcategoryId=form.subcategory.data, rate=rate)
+        db.session.add(new_rule)
+        db.session.commit()
+        flash("La nouvelle règle a été créée", "success")
+
+    except sa.exc.IntegrityError as err:
+        db.session.rollback()
+        flash(Markup("Erreur dans la création de la règle : {message}").format(message=str(err)), "warning")
+
+    return redirect(url_for("backoffice_web.offer_edition_rules.list_rules"), code=303)
+
+
+@offer_edition_rules_blueprint.route("/<int:rule_id>/delete", methods=["GET"])
+def get_delete_offer_edition_rule_form(rule_id: int) -> utils.BackofficeResponse:
+    rule_to_delete = offers_models.OfferEditionRule.query.filter_by(id=rule_id).one_or_none()
+    if not rule_to_delete:
+        raise NotFound()
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=empty_forms.EmptyForm(),
+        dst=url_for("backoffice_web.offer_edition_rules.delete_rule", rule_id=rule_id),
+        div_id=f"delete-offer-edition-rule-{rule_id}",  # must be consistent with parameter passed to build_lazy_modal
+        title="Supprimer une règle de modification d'offre",
+        button_text="Supprimer la règle",
+        information=Markup(
+            "La règle sur la sous-catégorie {subcategory} sera définitivement supprimée de la base de données. Veuillez confirmer ce choix."
```

Comment:
Détail : comme dans les messages flash, je trouverais plus clair d'afficher les noms variables en gras : 
```suggestion
            "La règle sur la sous-catégorie <b>{subcategory}</b> sera définitivement supprimée de la base de données. Veuillez confirmer ce choix."
```
Aussi dans le message flash en-dessous, par homogénéité avec les autres.